lib/colorado_lottery.rb

```diff
@@ -0,0 +1,34 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants, :winners, :current_contestants
+  def initialize()
```

Comment:
No parentheses needed here. Parentheses are gross. Frontend would disagree, but then again they actually enjoy writing Javascript, so they aren't to be trusted. We don't do parentheses when we don't pass arguments in ruby, because we're not extra like Frontend.